docs/components/card/README.md

```diff
@@ -120,10 +119,13 @@ CdrCards should always be used to link to other content, and the `cdr-card__link
 
 ## Accessibility
 
-To ensure that usage of this component complies with accessibility guidelines, do the following:
+Many WCAG requierments are contextual to their implementation. To ensure that usage of this component complies with accessibility guidelines you are responsible for the following:
 
-- TODO: something
-- TODO: something else
+- Use an `<article>` tag if the content includes a heading AND the content would be as useful on another web page
```

Comment:
i think we should clarify here that we mean that tag prop, so that people don't wrap the card content